genetics.pde

```diff
@@ -89,7 +93,7 @@ void tickPopulation() {
       Iterator foodIt = food.iterator();
       while (foodIt.hasNext()) {
         int[] coord = (int[]) foodIt.next();
-        if (distance(coord[0], coord[1], c.x, c.y)< 5) {
+        if (distance(coord[0], coord[1], c.x, c.y)< critterSize/2) { //chagned eat/drink distance to critterSize / 2
```

Comment:
Since your changes preserve a Critter constructor with a parameter for a custom size, this line shouldn't reference the constant size, but c.size, the size of the individual critter in question

Reply:
Since you want it to touch the food, it should be half the critter's radius plus half the food radius, or `(c.size+FOOD_SIZE)/2`
FOOD_SIZE does not yet exist but it would be a welcome change (see drawResources() - it's 5)